pinot-plugins/pinot-file-system/pinot-gcs/src/main/java/org/apache/pinot/plugin/filesystem/GcsPinotFS.java

```diff
@@ -130,7 +131,7 @@ private String sanitizePath(String path) {
 
   private URI getBase(URI uri) throws IOException {
     try {
-      return new URI(uri.getScheme(), uri.getHost(), null, null);
+      return new URI(SCHEME, uri.getHost(), DELIMITER, null);
```

Comment:
Is the third argument `path`? It is a little bit confusing to put delimiter as the path

Reply:
Without this we get a uri creation error, this is done to guarantee that we use the root directory, i.e. `gs://my_bucket/`, lmk what you think.